test/scenarios/el92-src@rpm-install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# shellcheck source=test/bin/get_rel_version_repo.sh
+source "${SCRIPTDIR}/get_rel_version_repo.sh"
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# The RPM-based system won't run greenboot automatically on startup,
+# so tell the framework not to wait for that.
+export SKIP_GREENBOOT=true
```

Comment:
Greenboot is installed as a dependency of MicroShift, so it's present.
The defaul preset for greenboot-healthcheck service seems to be `enabled`

```
$ systemctl status greenboot-healthcheck.service 
● greenboot-healthcheck.service - greenboot Health Checks Runner
     Loaded: loaded (/usr/lib/systemd/system/greenboot-healthcheck.service; enabled; preset: enabled)
    Drop-In: /etc/systemd/system/greenboot-healthcheck.service.d
             └─overrides.conf
     Active: active (exited) since Tue 2024-01-02 14:06:44 UTC; 2 days ago
   Main PID: 823 (code=exited, status=0/SUCCESS)
        CPU: 77ms
```

Reply:
The VM boots into an image without either package installed, then the tests install MicroShift. I'll clarify the comment.